config-linux.md

```diff
@@ -196,6 +196,39 @@ For example, to run a new process in an existing container without updating limi
 
 Runtimes MAY attach the container process to additional cgroup controllers beyond those necessary to fulfill the `resources` settings.
 
+### Cgroup ownership
+
+Runtimes MAY change (or cause to be changed) the owner of the
+container's cgroup to the host uid that maps to uid 0 in the
+container's user namespace, according to the following rules.
+
+Runtimes MUST NOT change the ownership of container cgroups when
+cgroups v1 is in use.  Cgroup delegation is not secure in cgroups
+v1.
```

Comment:
This should be a SHOULD NOT imho. While it might be a bad idea for security reasons, making this be something the spec explicitly disallows is a bit too extreme -- SHOULD NOT is strong enough, and including a justification is sufficient. There are reasons why you might want to do something different, and explicitly not allowing it a bit too extreme (not to mention adding MUST NOTs like this could be argued to be a SemVer-breaking change).

Reply:
relaxed to SHOULD.